LASTA-OUTDOOR/lasta
# Community : add friends

## Features
- [x] Link the connectivity state with andrew's repository (it nows detect whether you are online or not)
- [x] Create UI for adding friends
- [x] Create UI for accepting friends
- [x] Backend: Add friend
- [x] Backend: Accept friends
- [x] Test the view model
- [x] Test the UI

## How to test it yourself
- Go to the Friends tab in the community part
- Click on the + icon
- Write an email that is registered in the app (For testing purposes, you can add yourself as a friend)
- Submit the request
- Go back to the community tab, then click on the bell
- Accept or decline the invitation
- See the result in the friends tab

## Improvement needed
- Adding friends : autocompletion -> We have to write an exacte email which is not really a good feeling 
- Notification system : when sending a friend request, also trigger a notification system to the receiver
- Weird animation when using the navcontroller are ugly
